### PR TITLE
Added server/main.cpp to ignore list

### PR DESCRIPTION
**Pull request by Higami69**
#694

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
Looks good for ignoring that one file. But what I think should be added at the start of the file checking loop is that the path doesn't contain "ThirdParty" that should prevent false matches from random files in the dependencies (that might contains `src` in their paths).

**Comment by Higami69** [comment]
So if it has ThirdParty in the path it does need to get checked right?

**Comment by hhyyrylainen** [comment]
Yes

File: RunCodeFormatting.rb [hunk 1]
```diff
@@ -70,7 +70,7 @@
       raise e
     end
 
-    if path !~ /\/src\//i && path !~ /\/test\//i ||
+    if path !~ /\/src\//i && path !~ /\/test\//i  && path !~ /thirdparty/i ||
```
Comment (hhyyrylainen) [inline RunCodeFormatting.rb]: There is a typo here and missing capitalization, Linux is case sensitive. And this should be right at the top of the loop not here

Comment (Higami69) [reply RunCodeFormatting.rb]: what exactly do you mean with the top of the loop
as in only files that have "ThirdParty" in their path get checked?
or they don't get checked

Comment (hhyyrylainen) [reply RunCodeFormatting.rb]: There should be a check like `path =~ /\/ThirdParty/i` after line 22 with a `next` in it to skip all future handling for everything in ThirdParty

**Pull request merged**